packages/kcms/src/match/service/createRunResult.ts

```diff
@@ -0,0 +1,35 @@
+import { Result } from '@mikuroxina/mini-fn';
+import { SnowflakeIDGenerator } from '../../id/main';
+import { TeamID } from '../../team/models/team';
+import { FinishState, RunResult } from '../model/runResult';
+
+/**
+ * 走行結果を作成
+ */
+type MatchResults = {
+  teamID: TeamID;
+  points: number;
+  goalTimeSeconds: number;
+  finishState: FinishState;
+}[];
+export class CreateRunResultService {
+  constructor(private readonly idGenerator: SnowflakeIDGenerator) {}
+  async handle(matchResults: MatchResults): Promise<Result.Err<Error> | Result.Ok<RunResult[]>> {
```

Comment:
Result.Result<E, T>を使うべきでしょう
```suggestion
  async handle(matchResults: MatchResults): Promise<Result.Result<Error, RunResult[]>> {
```